tests/unit/package_managers/bundler/test_main.py

```diff
@@ -114,3 +115,83 @@ def test_get_main_package_name_and_version_from_repo_without_origin(
         _get_main_package_name_and_version(package_dir=rooted_tmp_path_repo, dependencies=[])
 
     assert "Failed to extract package name from origin remote" in exc_info.value.friendly_msg()
+
+
+def test__prepare_for_hermetic_build_injects_necessary_variable_into_empty_config(
```

Comment:
what's the double underscore for?

Reply:
One underscore to separate 'test' prefix from the name of the function, another one comes from the function name.